scripts/generateJson.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
The validate-json workflow should be updated to call this script otherwise the ios & android copies of the JSON file can fall out of sync